src/strategies/ethtrader-gov-weight/index.ts

```diff
@@ -0,0 +1,25 @@
+import fetch from 'cross-fetch';
+
+export const author = 'kohrts';
+export const version = '0.1.0';
+
+export async function strategy(space, network, provider, addresses, options, snapshot) {
+  const ETHTRADER_USERS_URL = 'https://raw.githubusercontent.com/EthTrader/donut.distribution/main/docs/users.json'
+  
+  const response = await fetch(ETHTRADER_USERS_URL, {
```

Comment:
"snapshot" should be used here, you don't just need voting power, you need voting power at a specific snapshot / time